### PR TITLE
Fix custom report generation

### PR DESCRIPTION
**Pull request by lucernae**
Let's see if we have any test fails

### CODE REVIEW

File: tasks/analysis.py [hunk 1]
```diff
@@ -419,24 +419,26 @@ def process_impact_result(self, impact_result, analysis_id):
...
-                impact_result['output']['analysis_summary']))
-            template_path = get_impact_path(os.path.join(
-                settings.GEOSAFE_IMPACT_OUTPUT_DIRECTORY, dirname, filename))
-            if os.path.exists(os.path.dirname(template_path)) and not (
-                    os.path.exists(template_path)):
-                shutil.copy(setting_template_path, template_path)
-                custom_template_path = template_path
+            filename = os.path.basename(analysis.custom_template)
+            dirname = os.path.dirname(
+                impact_result['output']['analysis_summary'])
```
Comment (myarjunar) [inline tasks/analysis.py]: `impact_url` is enough I reckon. It's been defined before

**Review by myarjunar — APPROVED** [review]

File: tasks/analysis.py (hunk 1 above)
Comment (lucernae) [reply tasks/analysis.py]: It's not url anymore, so we should change that.
Yeah, we can change into that.

Comment (lucernae) [reply tasks/analysis.py]: Uh, how do we change a suggestion? You have missing ) and I would like to put the line up one line.

Comment (myarjunar) [reply tasks/analysis.py]: there is no multiline support yet :joy:

**Pull request merged**